packages/@tinacms/graphql/src/database/index.ts

```diff
@@ -483,7 +483,10 @@ export class Database {
             cursor: btoa(edge.cursor),
           }
         } catch (error) {
-          if (error instanceof Error) {
+          if (
+            error instanceof Error &&
+            !edge.path.includes('.tina/__generated__/_graphql.json')
```

Comment:
@logan-anderson is this change related to the sorting feature?

Reply:
Sorta. I was getting a strange error message when an invalid sort key was given. This changes to just the raw error message